irrep/__gvectors.py

```diff
@@ -62,11 +62,26 @@ def calc_gvectors(
     igp = np.zeros(3)
     igall = []
     Eg = []
+    flag = True
     for N in range(nplanemax):
         if N % 10 == 0:
+            memory = np.full(10, True)
```

Comment:
a more elegant way would be to initialize memory only in the beginning and on every iteration of N to do smth like 
```
memory[:-1]=memory[1:]
memory[-1]=...
```
Thus the befaviour will be independent of N%10

Reply:
I see. We would also be saving time from some iterations, since the exception would arise as soon as 10 "fails" are found, not when a whole block of 10 "fails" is found. Thank you!